src/windows/win-crowdstrike-fix-bootloop-v2.ps1

```diff
@@ -30,6 +75,11 @@ forEach ( $partition in $partitionlist )
                 Log-Error "Load registry hive $regKey from $regFile failed with error: $result"
             } else {
                 Log-Info "Load registry hive $regKey from $regFile succeeded with message: $result"
+
+                if ($regKey -eq "HKLM\temp_software_hive_$guidSuffix") {
```

Comment:
update the script description to mention this step?

Reply:
Done